demos/nextAuth-demo/netlify.toml

```diff
@@ -0,0 +1,18 @@
+[build]
+command = "next build"
+publish = ".next"
+ignore = "git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF . ../../plugin"
+
+[build.environment]
+TERM = "xterm"
+NODE_VERSION = "17"
```

Comment:
[pebble] Node.js 17 is not an LTS version. Consider an LTS version, e.g. 16. As well, if this is set here, a corresponding [engines version](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines) should be set in package.json.

Reply:
Node 17 began to be specified within the demo projects as part of the changes to support [npm workspaces](https://github.com/netlify/netlify-plugin-nextjs/pull/1304/files#r847648971) within the project, and the use of an LTS version was brought up as part of the review. 

At the time we deferred the specification of an LTS version until v18 was released (which it now has been) because we weren't going to be waiting that much longer from the time of those changes, but it's likely worth doing all in one go as part of one set of changes rather than making all of those changes within the context of this pull request.

Will make the change to the `engines` version in the meantime though 👍🏻

Reply:
Mentioned offline but have created an issue tracking the upgrade to Node 18 for all the demo sites within this project so will consider this comment resolved for now